client/core/notification.go

```diff
@@ -100,19 +100,19 @@ type Notification interface {
 // FeePaymentNote is a notification regarding registration fee payment.
 type FeePaymentNote struct {
 	db.Notification
-	ConfirmationsRequired uint32 `json:"confirmationsrequired,omitempty"`
-	Confirmations         uint32 `json:"confirmations,omitempty"`
+	Confirmations uint32 `json:"confirmations,omitempty"`
+	DexURL        string `json:"dexurl,omitempty"`
 }
 
-func newFeePaymentNote(subject, details string, severity db.Severity) *FeePaymentNote {
+func newFeePaymentNote(subject, details string, severity db.Severity, url string) *FeePaymentNote {
```

Comment:
~~`url`~~ 🡒 `dexAddr` a couple of places. We're still a little mixed, but I don't wan't to make more work for #374